concepts/05 Widgets/Scheduler/030 Appointments/015 Appointment Types/020 All-Day Appointments.md

```diff
@@ -118,8 +121,6 @@ If appointments in your data source omit the **allDay** field but have another f
 
 ---
 
-[note]The **Scheduler** ignores the **startDate**'s and **endDate**'s time component for all-day appointments.
-
 To mark an appointment as all-day in the UI, toggle the *"All day"* switcher on the appointment details form. This form appears when a user [adds](/concepts/05%20Widgets/Scheduler/030%20Appointments/020%20Add%20Appointments '/Documentation/Guide/Widgets/Scheduler/Appointments/Add_Appointments/') or [updates](/concepts/05%20Widgets/Scheduler/030%20Appointments/030%20Update%20Appointments '/Documentation/Guide/Widgets/Scheduler/Appointments/Update_Appointments/') an appointment. 
 
 ![Scheduler All Day Appointment Details](/images/UiWidgets/Scheduler_AllDayAppointment_Details.png)
```

Comment:
Since I'm unable to add a comment to line 128 (below), can you please make the following changes?

1) `All-day appointments are displayed in the All-day panel, which is visible by default.`
I'd like to rewrite this sentence without using "by default". What do you think about the following?

`All-day appointments in the All-day panel are visible (the default setting).`

2) `If you are not using such appointments, hide the All-day panel along with all-day appointments by assigning **false** to the [showAllDayPanel](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/showAllDayPanel.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel') option.`

I don't think the first part of the sentence is necessary, because I think it's assumed that you would want to hide appointments if you're not using them. What do you think about the following suggestion?

`To hide the All-day panel along with all-day appointments, set the [showAllDayPanel](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/showAllDayPanel.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel') option to **false**.`

Reply:
Hi Albert,

> All-day appointments are displayed in the All-day panel, which is visible by default.

The phrase "which is visible by default" is about the panel, not the appointments. The whole paragraph is supposed to describe how to hide the all-day panel when there are no all-day appointments. How about the following version?

`All-day appointments are displayed in the All-day panel. If you do not use such appointments, disable the [showAllDayPanel](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/showAllDayPanel.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel') property to hide the All-day panel.`

Reply:
Hi Roman. Thank you for the clarification. Please apply your suggestion.